tracer/src/Datadog.Trace/Sampling/TriggerSamplingDecision.cs

```diff
@@ -0,0 +1,26 @@
+﻿// <copyright file="TriggerSamplingDecision.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+namespace Datadog.Trace.Sampling;
+
+internal enum TriggerSamplingDecision
+{
+    /// <summary>
+    /// Do not trigger a sampling decision.
+    /// </summary>
+    None,
+
+    /// <summary>
+    /// Trigger a sampling decision only if the sampling priority is not set.
+    /// </summary>
+    IfNotSet,
+
+    /// <summary>
+    /// Always trigger a sampling decision.
```

Comment:
```suggestion
    /// Always trigger a sampling decision, overwriting any previous sampling decision.
```

Reply:
Good one, thanks. I thought about renaming the enum member from `Always` to `Force` or even `Overwrite` to make that clear, BUT...

I may just remove that value since we're not using it anywhere. I think it would also mess up the rate limited since it would count the same trace more than once.

And if I do remove it, I may change the parameters to Booleans instead: `bool triggerSamplingDecision = false`.